studybuddies/src/app/app.component.ts

```diff
@@ -1,10 +1,47 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { User } from './models/user';
+import { AuthService } from './services/auth.service';
+import { TokenStorageService } from './services/token-storage.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
import { TokenStorageService } from "./services/token-storage.service";
```